upgrade/Stage_1.md

```diff
@@ -83,4 +83,7 @@ For any typescripts that were started during this stage, stop them with the `exi
 
 ## Stage completed
 
-This stage is completed. Continue to [Stage 2](Stage_2.md).
+This stage is completed. There are two different upgrade paths moving forward:
+
+- Path 1: For CSM only upgrade without IUF, continue to [Stage 2](Stage_2.md).
+- Path 2: For [upgrade CSM and additional products with IUF](../operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md), press the Back button in your web browser and perform the next step.
```

Comment:
What is the intended path after going back in the web browser? Can we be more specific about what stage to continue with.

Reply:
As the steps may change in the iuf workflow, I don't want to specify the next step here by have made an change by following the language used in https://cray-hpe.github.io/docs-csm/en-15/operations/iuf/workflows/image_preparation/